examples/spineml_braitenberg/experiment1.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><SpineML xmlns:UL="http://www.shef.ac.uk/SpineMLNetworkLayer" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.shef.ac.uk/SpineMLExperimentLayer" xsi:schemaLocation="http://www.shef.ac.uk/SpineMLNetworkLayer SpineMLNetworkLayer.xsd http://www.shef.ac.uk/SpineMLExperimentLayer SpineMLExperimentLayer.xsd"><Experiment name="robot" description="add a description for the experiment here"><Model network_layer_url="model.xml"/><Simulation duration="1" preferred_simulator="BRAHMS"><EulerIntegration dt="0.1"/></Simulation><ExternalInput target="LeftSensor" port="in" name="sensor left" tcp_port="50091" size="1" command="" host="0.0.0.0" timestep="0"/><ExternalInput target="RightSensor" port="in" name="sensor right" tcp_port="50091" size="1" command="" host="0.0.0.0" timestep="0"/><LogOutput name="steer left" target="SteerLeft" port="out" start_time="0" end_time="1e+8" tcp_port="50091" size="1" command="" host="0.0.0.0" timestep="0"/><LogOutput name="steer right" target="SteerRight" port="out" start_time="0" end_time="1e+8" tcp_port="50091" size="1" command="" host="0.0.0.0" timestep="0"/></Experiment></SpineML>
```

Comment:
Maybe fix the whitespace in these files with your IDE so they're a bit more readable

Reply:
Ask @ajc158  about that one - for some reason, while all the other SpineML creator-created xml has line endings - experiments don't